src/components/EmployeeList/EmployeeList.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from "react";
+
+import  { AppContext } from "../../Context/AppContext"
+
+const EmployeeList = () => {
+
+    const { employees } = useContext(AppContext);
+
+    return (
+        <div>
+            <ul>
+                {employees.map((user) => (
```

Comment:
if u are using - user.id, user.firstName, user.lastName you could destructure this object